### PR TITLE
made it pop

### LINKED ISSUE #3: Styling does not "pop" enough.
@hannahhch This page looks great, but I think we need more vibrant colors. It should be painful on the eyes. We need WAY higher contrast. More saturation. Broadway fonts. 

Screen Shot 2020-02-04 at 3 47 23 PM

Can we work on this?

### PR DESCRIPTION
**Pull request by hannahhch**
#### What's this PR do?
It updates the colors to be more bright. 

#### Where should the reviewer start?
Look in the css file. 

#### How should this be manually tested?
Open up index.html in your browser

#### What are the relevant tickets?
close #3 

#### Questions:
Are your eyes bleeding yet?

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -13,12 +14,13 @@ header {
 }
 
 section {
-  background-color: lightgrey;
+  background-color: #12E5DB;
```
Comment (khalidwilliams) [inline styles.css]: @hannahhch this change looks good, but it's creating a merge conflict with master. Please address and push back up.

File: styles.css [hunk 2]
```diff
@@ -1,4 +1,5 @@
 body {
+  background-color: #FF69B4;
```
Comment (khalidwilliams) [inline styles.css]: Love this change!

File: styles.css (hunk 1 above)
Comment (hannahhch) [reply styles.css]: Sorry about that, someone changed master that was weird! All good now. @khalidwilliams  can you give it another look and make sure its good to go.

Comment (khalidwilliams) [reply styles.css]: All good! I love the changes. Thanks for addressing that issue.

File: styles.css [hunk 3]
```diff
@@ -14,12 +14,13 @@ header {
 }
 
 section {
-  background-color: #4666FF;
+  background-color: #12E5DB;
```
Comment (Karlfunhouse) [inline styles.css]: BRIGHTER!!! MORE EYE BLOOD!!!

File: styles.css [hunk 4]
```diff
@@ -14,12 +14,13 @@ header {
...
   margin: 30px;
   padding: 20px;
 }
 
 h1, h2 {
+  color: #00EE0F;
```
Comment (dreardon1021) [inline styles.css]: Sick Color Bro

File: styles.css (hunk 2 above)
Comment (infamouskeyduster) [reply styles.css]: Teal for days! Hooray!

File: styles.css [hunk 5]
```diff
@@ -14,12 +14,13 @@ header {
 }
 
 section {
-  background-color: #4666FF;
+  background-color: #12E5DB;
   margin: 30px;
```
Comment (lanehunter-dev) [inline styles.css]: who needs margin? Let's get rid of this

File: styles.css (hunk 3 above)
Comment (MandyAlford) [inline styles.css]: I think we could go even bolder and brighter!

Comment (megan-venetianer) [inline styles.css]: This color is beautiful! Great change!

File: styles.css [hunk 6]
```diff
@@ -14,12 +14,13 @@ header {
...
   padding: 20px;
 }
 
 h1, h2 {
+  color: #00EE0F;
   text-align: center;
   font-family: 'Fantasy', serif;
```
Comment (Karlfunhouse) [inline styles.css]: Probably gonna need a little bit Fancier of a font.

File: styles.css (hunk 3 above)
Comment (cjrpostma) [inline styles.css]: It should have a bright red border too. And pirate cat gifs.

Comment (madisonrandle) [inline styles.css]: Love the color change! Can hardly look at the page with the colors together.

File: styles.css (hunk 6 above)
Comment (megan-venetianer) [inline styles.css]: Not a fan of this font...

File: styles.css (hunk 2 above)
Comment (KevDev90) [reply styles.css]: Dope colors braj!

File: styles.css (hunk 6 above)
Comment (KmBarnett) [inline styles.css]: Lets Pump Up the Font Size

File: styles.css (hunk 3 above)
Comment (edlsto) [inline styles.css]: What about purple??

File: styles.css [hunk 7]
```diff
@@ -14,12 +14,13 @@ header {
...
 section {
-  background-color: #4666FF;
+  background-color: #12E5DB;
   margin: 30px;
   padding: 20px;
 }
 
 h1, h2 {
+  color: #00EE0F;
   text-align: center;
```
Comment (hborrelli1) [inline styles.css]: I think increasing the font size by 100px, would make it easier to read.

File: styles.css (hunk 3 above)
Comment (infamouskeyduster) [reply styles.css]: I have been temporarily blinded ☝︎😎

File: styles.css [hunk 8]
```diff
@@ -14,12 +14,13 @@ header {
 }
 
 section {
-  background-color: #4666FF;
+  background-color: #12E5DB;
   margin: 30px;
   padding: 20px;
```
Comment (J-Poulter) [inline styles.css]: Needs more padding plz

File: styles.css [hunk 9]
```diff
@@ -14,12 +14,13 @@ header {
 }
```
Comment (jonathan-tschida) [inline styles.css]: Needs more whitespace here, about 300 more returns should do it

File: styles.css [hunk 10]
```diff
@@ -14,12 +14,13 @@ header {
...
 
 section {
-  background-color: #4666FF;
+  background-color: #12E5DB;
   margin: 30px;
   padding: 20px;
 }
```
Comment (KmBarnett) [inline styles.css]: I think it would POP more with some major box shadow make it fly off the page

File: styles.css (hunk 4 above)
Comment (edlsto) [inline styles.css]: Honestly I think yellow would look better.

Comment (madisonrandle) [inline styles.css]: Not sure I love the watermelon vibe, however I do appreciate the brightness of this scheme.

File: styles.css (hunk 8 above)
Comment (Trond240) [reply styles.css]: This color is dope but lets get a little darker!

File: styles.css (hunk 2 above)
Comment (KareNguyen94) [reply styles.css]: Don't think the teal is that great, can we we please change it to papayaWhip?

File: styles.css (hunk 6 above)
Comment (kimalajoy) [inline styles.css]: Maybe this font color should be a different color, too. That would enhance the eye blood possibilities

File: styles.css (hunk 3 above)
Comment (TimNguyen21) [inline styles.css]: I'm not feeling this color for the section background.  Maybe use this color: #53F607 instead?

File: styles.css (hunk 6 above)
Comment (tanishalatoya) [inline styles.css]: I think this font is too "fancy." How about toning it down to Arial or Open Sans, so something similar?

**Comment by jordanwa1947** [comment]
hmmm

File: styles.css (hunk 10 above)
Comment (Trond240) [reply styles.css]: A little more margin please

File: styles.css [hunk 11]
```diff
@@ -14,12 +14,13 @@ header {
...
```
Comment (ERaith) [inline styles.css]: Remove all margins! We need wall to wall text!

**Pull request merged**